README.md

```diff
@@ -151,7 +151,7 @@ A/B testing, canary release, blue-green deployment, limit rate, defense against
 
 - **Multi-Language support**
   - Apache APISIX is a multi-language gateway for plugin development and provides support via `RPC` and `Wasm`.
-  ![Multi Language Support into Apache APISIX](docs/assets/images/apisix-multi-lang-support.png)
```

Comment:
We can remove the old image?
https://github.com/apache/apisix/blob/master/docs/assets/images/apisix-multi-lang-support.png

Reply:
@membphis and I think that the old image may be used in other existing websites, so that its' deletion may cause panic. We'd better remain it temporarily.

Reply:
What does "temporarily" mean? A month or a year?

Reply:
If panic, then we fix it one by one.